examples/bandgap/bandgap_demo.ipynb

```diff
@@ -1,1001 +1,637 @@
 {
- "cells": [
-  {
-   "cell_type": "code",
-   "execution_count": 2,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "%matplotlib inline\n",
-    "\n",
-    "from sklearn.model_selection import cross_val_score, cross_val_predict, GridSearchCV, ShuffleSplit, KFold\n",
-    "from matminer.featurizers.conversions import StrToComposition\n",
-    "from matminer.featurizers.base import MultipleFeaturizer\n",
-    "from matminer.featurizers import composition as cf\n",
-    "from sklearn.ensemble import RandomForestRegressor\n",
-    "from matplotlib.colors import LogNorm\n",
-    "from matplotlib import pyplot as plt\n",
-    "from sklearn import metrics\n",
-    "import pandas as pd\n",
-    "import numpy as np\n",
-    "import warnings\n",
-    "import glob\n",
-    "\n",
-    "warnings.filterwarnings('ignore')"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 3,
-   "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "=== Fetching Data Package _test_foundry_oqmd_bandgap_v1.1 ===\n",
-      "=== Fetching Data Package _test_foundry_mp_bandgap_v1.1 ===\n",
-      "=== Fetching Data Package _test_foundry_experimental_bandgap_v1.1 ===\n",
-      "=== Fetching Data Package _test_foundry_assorted_computational_bandgap_v1.1 ===\n"
-     ]
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "kernelspec": {
+      "display_name": "Python 3",
+      "language": "python",
+      "name": "python3"
+    },
+    "language_info": {
```

Comment:
Comment for `dendrite_segmentation` notebook because Github won't let me comment on it directly (diff too big)
---------------------------------------------------------------

I'd add one sentence to the *Create Metadata* section, maybe just:

Create the metadata for this dataset so you can push it to Foundry if you like (see other notebooks for dataset publication).